packages/cms/src/plugins/sanity-deploy-button/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+
+import Button from 'part:@sanity/components/buttons/default';
+import { useSecrets, SettingsView } from 'sanity-secrets';
+
+import styles from './deploy.css';
+
+const namespace = 'github-actions-deployment';
+
+const pluginConfigKeys = [
+  {
+    key: 'webhookUrl',
+    description: 'The webhook to trigger a deployment',
+    title: 'The webhook to trigger a deployment',
+  },
+];
+
+type Secrets =
+  | {
+      webhookUrl?: string;
+    }
+  | undefined;
+
+function Deploy() {
+  const { secrets }: { secrets: Secrets } = useSecrets(namespace);
+  const [showSettings, setShowSettings] = useState(false);
+
+  // useEffect(() => {
+  //   if (!secrets) {
+  //     setShowSettings(true);
+  //   }
+  // }, [secrets]);
+
+  function triggerDeploy() {
+    if (secrets?.webhookUrl) {
+      fetch(secrets.webhookUrl, {
+        mode: 'no-cors',
+      });
+    } else {
+      console.error(
+        'The studio is missing the webhookUrl secret. Check your configuration'
+      );
+    }
+  }
+
+  return (
+    <div className={styles.container}>
+      <header className={styles.header}>
+        <h2 className={styles.title}>Deploy to production</h2>
```

Comment:
staging or production?
```suggestion
        <h2 className={styles.title}>Deploy to staging</h2>
```